apps/docs/content/components/Grid.mdx

```diff
@@ -0,0 +1,274 @@
+---
+title: Grid
+status: Experimental
+source: https://github.com/primer/brand/tree/main/packages/react/src/Grid/Grid.tsx
+storybook: '/brand/storybook/?path=/story/components-grid--default'
+description: Use Grid to create flexible and responsive grid-based layouts
+---
+
+import {Label} from '@primer/react'
+import {PropTableValues} from '../../src/components'
+
+```js
+import {Grid} from '@primer/react-brand'
+```
+
+## Examples
+
+### Default
+
+The `Grid` component uses a 12-column layout system. Use `span` to control the number of columns that should be displayed.
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column span={1}></Grid.Column>
+  <Grid.Column span={11}></Grid.Column>
+  <Grid.Column span={2}></Grid.Column>
+  <Grid.Column span={10}></Grid.Column>
+  <Grid.Column span={3}></Grid.Column>
+  <Grid.Column span={9}></Grid.Column>
+  <Grid.Column span={4}></Grid.Column>
+  <Grid.Column span={8}></Grid.Column>
+  <Grid.Column span={5}></Grid.Column>
+  <Grid.Column span={7}></Grid.Column>
+  <Grid.Column span={6}></Grid.Column>
+  <Grid.Column span={6}></Grid.Column>
+  <Grid.Column span={7}></Grid.Column>
+  <Grid.Column span={5}></Grid.Column>
+  <Grid.Column span={8}></Grid.Column>
+  <Grid.Column span={4}></Grid.Column>
+  <Grid.Column span={9}></Grid.Column>
+  <Grid.Column span={3}></Grid.Column>
+  <Grid.Column span={10}></Grid.Column>
+  <Grid.Column span={2}></Grid.Column>
+</Grid>
+```
+
+### Nested grids
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column>
+    <Grid enableOverlay>
+      <Grid.Column span={4}></Grid.Column>
+      <Grid.Column span={4}></Grid.Column>
+      <Grid.Column span={4}></Grid.Column>
+    </Grid>
+  </Grid.Column>
+</Grid>
+```
+
+### Responsive behavior
+
+Use `start` with an `Object` of breakpoint-specific keys and `number` values to enable responsive behavior.
+
+```jsx live
+<Grid enableOverlay>
+  <Grid.Column
+    span={{xsmall: 12, small: 12, medium: 6, large: 4, xlarge: 3, xxlarge: 2}}
```

Comment:
Is/Will it be possible to set only a e.g. middle point where the span changes:

In Primer, `col-md-6` would set the column to be wide 12 units until the `md` breakpoint and upwards, so you don't have to specify `col-12 col-sm-12 col-md6 col-lg-6 col-xl-6`.

Reply:
Yeah, this is a reallly verbose example 😀. It's intended to work the way you described because we use the min-width media query. When you change the examples in docs, does it not behave that way right now for you?

Reply:
Great! It actually behaves as described when I change it in the docs.

Maybe we can update the examples with the shortest form in order to promote this practice?

Reply:
👍 .. Happy to update it, but wanted to clarify that it's intentionally verbose to show the full-breadth of supported breakpoints. The only other way to know the values, is to review the source code. I could add a note above this example, to indicate that we use min-width?

Reply:
Yes, fair point! Maybe we can add a note about the shorthand syntax or vice-verca?

Reply:
Added some additional copy about this.